### ISSUE TITLE
Newest release installed via PIP uses a 6.0.0-preview 

### ISSUE THREAD

**Issue by revchris** [microsoft/azure-devops-python-api#357]
Just had a user install azure-devops via "pip install azure-devops".  This apparently was updated yesterday, Aug 13, and is azure-devops 6.0.0b3.  When running the python script to get a connection we are receiving an error that we need to include "-preview" flag.
I don't see anyway to actually include that.

**Comment by tedchamb** [comment]
Thanks for reporting, I will take a look

**Comment by tedchamb** [comment]
@revchris , I'm not able to repro.  Are you hitting the hosted service, or a private instance of Azure DevOps Server?  If Azure DevOps Server, what version is it?

Can you set the `azure_devops_python_print_urls` env var to `true` and then rerun?  If you send me the list of urls that get logged (after you have removed any private data from them), it will help me determine where its failing.

**Comment by revchris** [comment]
going to the site, https://pypi.org/project/azure-devops/#files, shows the new package being downloaded.

**Comment by junyzheng** [comment]
running into the same issue today. if you check the file the api version looks like this: `version='6.0'` but i think it should be `version='6.0-preview'`

**Comment by jhw866** [comment]
I too am running into this issue. I get this error `The requested version "6.0" of the resource is under preview. The -preview flag must be supplied in the api-version for such requests. For example: "6.0-preview"`. I just installed the package today.

**Comment by remyjette** [comment]
here's my repo in a hosted Azure Pipelines job using Python 3.8.5 on `ubuntu-latest`:

```python
credentials = BasicTokenAuthentication({"access_token": environ["SYSTEM_ACCESSTOKEN"]})
connection = Connection(environ["SYSTEM_COLLECTIONURI"], creds=credentials)
identity_client = connection.clients.get_identity_client()
identity_client.read_identity(environ["BUILD_QUEUEDBYID"])
```

Previously this worked, now it fails with:

```
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/azure/devops/v6_0/identity/identity_client.py", line 225, in read_identity
    response = self._send(http_method='GET',
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/azure/devops/client.py", line 104, in _send
    response = self._send_request(request=request, headers=headers, content=content, media_type=media_type)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/azure/devops/client.py", line 68, in _send_request
    self._handle_error(request, response)
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/azure/devops/client.py", line 270, in _handle_error
    raise AzureDevOpsServiceError(wrapped_exception)
azure.devops.exceptions.AzureDevOpsServiceError: The requested version "6.0" of the resource is under preview. The -preview flag must be supplied in the api-version for such requests. For example: "6.0-preview"
```

It works again if I change
```python
identity_client = connection.clients.get_identity_client()
```
to
```python
identity_client = connection.clients_v5_1.get_identity_client()
```

**Comment by tedchamb** [comment]
Fixed with #358

**Comment by tedchamb** [comment]
Thanks for all of you for providing info on this issue.  It should now be fixed in [6.0.0b4](https://github.com/microsoft/azure-devops-python-api/releases/tag/6.0.0b4).  Sorry for the inconvenience.
